ramp-database/ramp_database/tools/leaderboard.py

```diff
@@ -377,21 +383,26 @@ def update_user_leaderboards(session, event_name, user_name):
         The event name.
     user_name : str
         The user name. If None, scores from all users will be queried.
+    pending_only : bool, default is False
+        Whether or not to update the whole leaderboards or only the pending
+        submissions. You can turn this option to True when adding a new
+        submission in the database.
```

Comment:
It's not fully clear (from reading this PR) what this exactly means. What are "pending" submissions? (those that not have been trained yet?)

Reply:
The new one only. We can change the name